src/grt/grt-wave_opt_file.adb

```diff
@@ -42,21 +43,27 @@ package body Grt.Wave_Opt_File is
          when Warning =>
             Report_C ("warning: ");
       end case;
-      Report_C ("in file '");
       Report_C (File_Path.all);
-      Report_C ("' at line ");
-      To_String (Lineno_Str, First, Ghdl_I32 (Line_Context.Num));
-      Report_C (Lineno_Str (First .. Lineno_Str'Last));
-      Report_C (" - ");
-      Report_C (Line_Context.Str.all);
-      Report_C (" : ");
+      Report_C (":");
+      To_String (Value_Str, First, Ghdl_I32 (Line_Pos));
+      Report_C (Value_Str (First .. Value_Str'Last));
+      Report_C (":");
+      To_String (Value_Str, First, Ghdl_I32 (Column_Pos));
```

Comment:
Fixed


Le 27. 07. 16 à 05:17, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file.adb 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72375088>:
>
> >        Report_C (File_Path.all);
> > -      Report_C ("' at line ");
> > -      To_String (Lineno_Str, First, Ghdl_I32 (Line_Context.Num));
> > -      Report_C (Lineno_Str (First .. Lineno_Str'Last));
> > -      Report_C (" - ");
> > -      Report_C (Line_Context.Str.all);
> > -      Report_C (" : ");
> > +      Report_C (":");
> > +      To_String (Value_Str, First, Ghdl_I32 (Line_Pos));
> > +      Report_C (Value_Str (First .. Value_Str'Last));
> > +      Report_C (":");
> > +      To_String (Value_Str, First, Ghdl_I32 (Column_Pos));
>
> Likewise.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/77c854621cce3da7afdd08837584aea3fe27c1ca#r72375088>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePhAKKOZ2MONI9nAohTMLZOXgKh8iks5qZs3TgaJpZM4JVgTD>.
>